lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoPsiDeclarationImpl.kt

```diff
@@ -57,6 +59,79 @@ internal open class KoPsiDeclarationImpl(private val psiElement: PsiElement) : K
 
     override fun hasKDoc() = kDoc != null
 
+    protected open fun verifyDescription(value: Boolean) = TagHelper.verifyDescription(value, kDoc)
+
+    protected open fun verifyParamTag(value: Boolean) = true
+
+    protected open fun verifyReturnTag(value: Boolean) = true
+
+    protected open fun verifyConstructorTag(value: Boolean) = true
+
+    protected open fun verifyReceiverTag(value: Boolean) = true
+
+    protected open fun verifyPropertyTag(value: Boolean) = true
+
+    protected open fun verifyThrowsTag(value: Boolean) = true
+
+    protected open fun verifyExceptionTag(value: Boolean) = true
+
+    protected open fun verifySampleTag(value: Boolean) = true
+
+    protected open fun verifySeeTag(value: Boolean) = true
+
+    protected open fun verifyAuthorTag(value: Boolean) = true
+
+    protected open fun verifySinceTag(value: Boolean) = true
+
+    protected open fun verifySuppressTag(value: Boolean) = true
+
+    protected open fun verifyVersionTag(value: Boolean) = true
+
+    protected open fun verifyPropertySetterTag(value: Boolean) = true
+
+    protected open fun verifyPropertyGetterTag(value: Boolean) = true
+
+    @Suppress("detekt.CyclomaticComplexMethod")
+    override fun hasCompleteKDoc(
```

Comment:
I think `Valid` word will be better after all. Let's rename this to `hasValidKDoc`